### PR TITLE
make case study suggestion support multi language

### PR DESCRIPTION
**Pull request by zirain**
before:
image

after:
image

### CODE REVIEW

File: i18n/zh.toml [hunk 1]
```diff
@@ -339,3 +339,6 @@ other = "更多案例"
 
 [go_to_solutions]
 other = "转到解决方案"
+
+[case_study_slogen]
+other = "阅读受益于实施 Istio 的公司的案例"
```
Comment (windsonsea) [inline i18n/zh.toml:344]:
```suggestion
other = "阅读因实施 Istio 而受益的公司案例"
```
这样会不会通顺点 😄

File: i18n/en.toml [hunk 2]
```diff
@@ -354,3 +354,6 @@ other = "Also used by"
 
 [go_to_solutions]
 other = "Go to solutions"
+
+[case_study_slogen]
```
Comment (windsonsea) [inline i18n/en.toml:358]:
```suggestion
[case_study_slogan]
```
slogan 貌似拼错了，需全局替换吧

**Review by windsonsea — APPROVED** [review]

**Review by Xunzhuo — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Pull request merged**